src/build/buildExecutor.ts

```diff
@@ -66,6 +80,35 @@ export class BuildExecutor {
         return buildResult;
     }
 
+    public startLanguageServer(input: BuildInput, buildUserToken: string) {
+        let buildParameters = this.getBuildParameters(undefined, input, buildUserToken);
+        if (buildUserToken) {
+            buildParameters.envs['DOCS_OPS_TOKEN'] = buildUserToken;
+        }
+        let command = this._binary;
+        let args = buildParameters.serveCommand;
+        let options = { env: buildParameters.envs, cwd: this._cwd };

```

Comment:
merge with the process environment to inherit the current process environment (reference the function `executeDocfx`)